### PR TITLE
added checks to multer and front

### PR DESCRIPTION
**Pull request by DarkOverkill**

### CODE REVIEW

File: dashboard/src/components/common/ImageUploadField.jsx [hunk 1]
```diff
@@ -49,6 +51,9 @@ export default class ImageUploadField extends React.Component {
     upload = e => {
       let formData = new FormData();
       const file = e.target.files[0];
+      if (!urlIsImage(file.name)) {
+        return toast('Wrong file extansion!');
```
Comment (NazarGorokhivskiy) [inline dashboard/src/components/common/ImageUploadField.jsx]: "Wrong file extension"

File: dashboard/server/middlewares/multerFileUpload.js [hunk 2]
```diff
@@ -9,6 +10,22 @@ const storage = multer.diskStorage({
...
+  fileFilter: function (req, file, cb) {
+    if (fileIsImage(file.originalname)) return cb(null, true);
+    cb(null, false);
+  },
+}).single('file');
+
+async function uploadFile(req, res, next) {
+  upload(req, res, function(err) {
+    if (err) {
+      return res.send({ error: `${err.message}. Max allowed file size is 2MB.` });
```
Comment (NazarGorokhivskiy) [inline dashboard/server/middlewares/multerFileUpload.js]: return res.status(400).send( ... );

Comment (DarkOverkill) [reply dashboard/server/middlewares/multerFileUpload.js]: done

File: dashboard/src/components/common/ImageUploadField.jsx (hunk 1 above)
Comment (DarkOverkill) [reply dashboard/src/components/common/ImageUploadField.jsx]: done

**Pull request merged**